arithmetization/src/test/java/net/consensys/linea/zktracer/module/oob/OobCallTest.java

```diff
@@ -98,8 +99,45 @@ void TestRecursiveCallsWithBytecode() {
     final Hub hub = bytecodeRunner.getHub();
 
     assertTrue(Exceptions.none(hub.pch().exceptions()));
+  }
+
+  /**
+   * Same as {@link #TestRecursiveCallsWithBytecode()} but with an ADD opcode at the end triggering
+   * SUX
+   */
+  @Test
+  void TestRecursiveCallsWithBytecodeFollowedByStackUnderflow() {
```

Comment:
Please rename tests to start with a lower case letter.

Reply:
Let's do it in a separate PR project-wise.